modules/setting/setting.go

```diff
@@ -916,6 +918,12 @@ func NewContext() {
 		}
 	}
 
+	if PprofDataPath != "" {
```

Comment:
this should be: `if EnablePprof {`

Reply:
Than just add code to create it in second case

Reply:
I have decided to create the directories in one place (currently in cmd/serve.go). Felt a little weird mkdir in setting and in serv. Check my last commit.